elements/lisk-chain/src/chain.ts

```diff
@@ -378,8 +359,7 @@ export class Chain {
 	}
 
 	public async resetState(): Promise<void> {
-		await this.storage.entities.Account.resetMemTables();
-		await this.storage.entities.ConsensusState.delete();
+		await this.dataAccess.resetMemTables();
```

Comment:
While we are at these could we remove the `Mem` part from names? It has always seem that something was exclusively in RAM memory but the truth is that it might or not depending on the storage backend.

Reply:
This function will be removed in https://github.com/LiskHQ/lisk-sdk/issues/5320